frontend/packages/ceph-storage-plugin/src/components/ocs-install/ocs-install.scss

```
@@ -25,3 +25,7 @@
   font-weight: var(--pf-global--FontWeight--bold);
   font-size: var(--pf-global--FontSize--md);
 }
+
+.ceph-install-select-mode {
```

lets stick to the convention.
```suggestion
.ceph-ocs-install-select-mode {
```

---

ack